qemux86-64.robot

```diff
@@ -41,6 +42,15 @@ Get host OS version of the image
   Set Suite Variable    ${os_version}   ${return_os_version}
   Unmount "${mount_destination}"
   Detach loop device "${path_to_loop}"
+Enable getty service on the image
```

Comment:
This could be made into a generic keyword that takes in the image path as a parameter.

Enable getty service on image "${image}"